### ISSUE TITLE
urbit-king rc binaries problematic on NixOS

### ISSUE THREAD

**Issue by jtobin** [urbit/urbit#3369]
@syzygyzer and I have both run into the following problem on recent tags, e.g. `kh-rc2`:

```
[jtobin@pollux:~/kh-rc2-linux64]$ ll
total 63840
-r-xr-xr-x 1 jtobin users 11703688 Aug 18 04:18 urbit
-r-xr-xr-x 1 jtobin users 42115056 Aug 18 04:27 urbit-king
-r-xr-xr-x 1 jtobin users 11549856 Aug 18 04:18 urbit-worker

[jtobin@pollux:~/kh-rc2-linux64]$ ./urbit-king
-bash: ./urbit-king: No such file or directory
```

/cc @pilfer-pandex

**Comment by pilfer-pandex** [comment]
@jtobin can you confirm that this is fixed on the new release/next-vere builds?

**Comment by jtobin** [comment]
@pilfer-pandex Have you got a link to any linux64 tgz'd release binaries from that branch?  IIRC things worked fine when I built from scratch, just not in the release binaries.  I checked for `$COMMIT_HASH_FROM_NEXT_VERE-linux64.tgz` on bootstrap.urbit.org but couldn't find anything -- I can't remember what way they're packaged up exactly.

In the meantime @joemfb claims that #3806 fixes this issue, so he's probably right.
